delegates.rb

```diff
@@ -234,31 +234,26 @@ def get_rights(image_id, ip)
   def fetch(path, ip)
     require 'net/http'
     require 'uri'
-    # logger = Java::edu.illinois.library.cantaloupe.delegate.Logger
-    #.logger.debug("API URL IS: #{api_url(path)}")
-    # logger.debug("IPS ARE: #{ip}")
-
+    require 'json'
+    
     uri = URI.parse(api_url(path))
+
     request = Net::HTTP::Post.new(uri)
-    request.body = `'{"ips":["#{ip}"]}'`
-    request.set_content_type("application/json")
-    request["Accept"] = "application/json"
-    request["Authorization"] = "Token token=#{Secret.api_configuration[:auth_token]}"
-    
-    req_options = {
-      use_ssl: uri.scheme == "https",
-    }
+    request.body = '{"ips":["#{ip}"]}'
+    request.content_type = 'application/json'
+    request['Accept'] = 'application/json'
+    request['Authorization'] = "Token token=#{Secret.api_configuration[:auth_token]}"
+
+    http = Net::HTTP.new(uri.hostname, uri.port)
+    http.use_ssl = true if uri.scheme == 'https' # which it should be, as I've hard-coded the api url to https.
+
+    response = http.request(request)
 
-    response = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
-      http.request(request)
-    end
-    
-    # logger.debug("RESPONSE IS: #{response}")
     return response.body
   end
 
   def api_url(path)
-    "#{Secret.api_configuration[:api_url]}/api/v2/#{path}"
+    "https://api.repo.nypl.org/api/v2/#{path}"

```

Comment:
How does this change the behavior if previously we had:

api_url: 'https://api.repo.nypl.org

in secrets.rb?

Reply:
I don't know. That's the thing. But if I hard-code it, it will rule out a botched env setting somewhere. And there's really no need for this to be an env setting.